bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/hob.xml

```diff
@@ -12,6 +12,7 @@
 
 		<label>Hob</label>
 		<description>This is a Miele@home compatible hob</description>
+		<semantic-equipment-tag>Cooktop</semantic-equipment-tag>
```

Comment:
It might be worth considering adding `Hob` as a synonym in https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv